geniuspaste/src/geniuspaste.c

```diff
@@ -94,6 +94,54 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+/* Windows compatibility for resolving system plugin data path.
+ * This only works if the plugin's data is installed in the same tree as the
+ * plugin DLL, which might not be true (as the DLL has to be in Geany's libdir
+ * but the plugin's data might be anywhere in theory) */
```

Comment:
@b4n GResource is fine for default configuration data, IMO. I have plugins where I just hard-code the default config into a C string, and on first use dump it into a file in the users config dir. Using GResource would be effectively the same. Having user-editable data files in a non-user accessible directory is not a good idea, which is what it sounds like this plugin is doing, instead of copying/writing them into a directory the user can access.